runtime/hydradx/src/evm/runner.rs

```diff
@@ -260,4 +260,56 @@ where
 			config,
 		)
 	}
+
+	fn create_force_address(
```

Comment:
Not sure who uses this, but EVM Runner requires this function.

Reply:
ok, i didnt found case yet, but creations of the contracts is restricted so we should make sure that we filter it accordingly if this would become avaliable for users somehow

i would currently just error out instead as "not supported" or something to be safe